helm_deploy/values-dev.yaml

```diff
@@ -1,18 +1,21 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-replicaCount: 2
+replicaCount: 1
```

Comment:
Perhaps put a comment against this why setting to 1??  not a big deal if none but could potentially be an issue if changed.

Reply:
Yes, can add that. 
This are per env values, that usually will be reviewed at different stages of the project.